projects/kit/components/combo-box/combo-box.component.ts

```diff
@@ -153,7 +153,7 @@ export class TuiComboBoxComponent<T>
     }
 
     get canOpen(): boolean {
```

Comment:
Since `interactive` is already a public getter we can just use it wherever we used `canOpen`.